native_words.asm

```diff
@@ -5893,8 +5863,7 @@ plus_loop_runtime:
                 adc 1,x         ; MSB of step
                 pha             ; put MSB of index back on stack
 
-                tya             ; put LSB of index back on stack
-                pha
+                phy             ; put LSB of index back on stack
```

Comment:
-1 byte with 65c02 inst

Reply:
There are probably lots more small 65C02 specific speedups in Tali, so if you see any others while looking through the source code, I'd be happy for a pull request even for something as small as this.